common/model-views.cpp

```diff
@@ -549,6 +549,17 @@ namespace rs2
         }
         return option;
     }
+    std::string option_model::get_option_description(rs2_option option, bool display_in_cm) const
+    {
+        std::string desc (endpoint->get_option_description(option));
+        std::string meters_str("meters");
+        if (display_in_cm)
+        {
+            auto pos = desc.find(meters_str);
+            desc.replace(pos, meters_str.size(), "cm");
```

Comment:
The heuristic will replace "millimeter" with "millicm"

Reply:
not relevant since the adjust_description has been implemented, according to other comment of yours